arkouda/numeric.py

```diff
@@ -422,16 +425,26 @@ def cos(pda: pdarray) -> pdarray:
     return create_pdarray(repMsg)
 
 
+def _hash_helper(a):
+    from arkouda import SegArray as _Segarray
+
+    # TODO move to JSON (see issue #2464), right now we '+' as a delimiter
+    if isinstance(a, _Segarray):
+        return f"{a.segments.name}+{a.values.name}+{a.values.objType}"
+    else:
+        return a.name
```

Comment:
We have the components to handle this now. Not for complex JSON, but for a simple case like this we can use `jsonToMap` in `GenSymIO.chpl` to convert to a map. I do think limiting our tech debt moving forward would be would be advisable.

Let me know if you need an example of how to do this with the list of JSON elements.

Reply:
awesome! ill add that in!